Library/Formula/exploitdb.rb

```diff
@@ -0,0 +1,22 @@
+class Exploitdb < Formula
+  desc "The official Exploit Database"
+  homepage "https://www.exploit-db.com/"
+  url "https://github.com/offensive-security/exploit-database.git",
+    :tag => "2016-03-21",
+    :revision => "e67a88eeffbc511d3e5e10bfc26639e6fce16576"
+
+  head "https://github.com/offensive-security/exploit-database.git"
+
+  def install
+    inreplace "searchsploit", "gitpath=\"/usr/share/exploitdb\"", "gitpath=\"#{pkgshare}\""
+    bin.install "searchsploit"
+
+    pkgshare.install "files.csv"
+    pkgshare.install "platforms"
+    pkgshare.install ".git"
```

Comment:
You can do everything in one line: `pkgshare.install %w[files.csv platforms .git]`.